src/core/Track.cpp

```diff
@@ -3042,3 +3043,30 @@ void TrackView::createTCOView( TrackContentObject * tco )
 	}
 	tco->selectViewOnCreate( false );
 }
+
+
+/*! \brief Show a message box warning the user that this track is about to be closed */
+bool TrackOperationsWidget::warnRemoveTrack()
+{
+    QString messageRemoveTrack = tr("After removing a track, it can not"
+                                    "be recovered. Are you sure you want to remove track \"");
+    messageRemoveTrack += m_trackView->getTrack()->name() + "\"?";
```

Comment:
I recommend using [`QString::arg()`](https://doc.qt.io/qt-5/qstring.html#arg) instead of manual concatenation. In some languages, that word order will make no sense when translated.